### PR TITLE
Fix initial json editor

### PR DESCRIPTION
**Pull request by abefernan**
Fixes not being able to run query or execute contract with the placeholder data

### CODE REVIEW

File: src/pages/contract/ExecuteContract.tsx [hunk 1]
```diff
@@ -75,9 +86,7 @@ export function ExecuteContract({ contractAddress }: Props): JSX.Element {
           <JSONInput
             width="100%"
             height="200px"
-            placeholder={{
-              transfer: { recipient: "cosmos1zk4hr47hlch274x28j32dgnhuyewqjrwxn4mvm", amount: "1" },
-            }}
+            placeholder={executePlaceholder}
```
Comment (webmaster128) [inline src/pages/contract/ExecuteContract.tsx:89]: Can we unset this for simplification? It should not be needed anymore, right?

File: src/pages/contract/ExecuteContract.tsx [hunk 2]
```diff
@@ -90,8 +99,7 @@ export function ExecuteContract({ contractAddress }: Props): JSX.Element {
           <JSONInput
             width="100%"
             height="120px"
-            // eslint-disable-next-line @typescript-eslint/camelcase
-            placeholder={[{ denom: settings.backend.denominations[0], amount: "1" }]}
+            placeholder={coinsPlaceholder}
```
Comment (webmaster128) [inline src/pages/contract/ExecuteContract.tsx:102]: same here

File: src/pages/contract/QueryContract.tsx [hunk 3]
```diff
@@ -56,8 +63,7 @@ export function QueryContract({ contractAddress }: Props): JSX.Element {
           <JSONInput
             width="100%"
             height="200px"
-            // eslint-disable-next-line @typescript-eslint/camelcase
-            placeholder={{ get_balance: { address: "cosmos1zk4hr47hlch274x28j32dgnhuyewqjrwxn4mvm" } }}
+            placeholder={queryPlaceholder}
```
Comment (webmaster128) [inline src/pages/contract/QueryContract.tsx:66]: and here

File: src/pages/contract/ExecuteContract.tsx (hunk 1 above)
Comment (abefernan) [reply src/pages/contract/ExecuteContract.tsx:89]: The placeholder is the text the user sees and can edit on the editor. If I remove the placehoder attribute the msgObject would be prefilled and you would be able to execute it, but you wouldn't see anything on the editor

**Review by webmaster128 — APPROVED** [review]

File: src/pages/contract/ExecuteContract.tsx (hunk 1 above)
Comment (webmaster128) [reply src/pages/contract/ExecuteContract.tsx:89]: Thanks for clarification

**Pull request merged**